govlearn-api/src/test/java/com/unimuenster/govlearnapi/course/controller/CourseCompletionControllerTest.java

```diff
@@ -34,25 +39,30 @@ void setUp() {
 
     @Test
     void addCourseCompletion() {
-        courseCompletionService.addCourseCompletion(authenticationService.getCurrentUser(), initializerService.getCourse1().getId());
+        courseCompletionService.addCourseCompletion(authenticationService.getCurrentUser(), initializerService.getCourse4().getId());
 
-        assertTrue(initializerService.getUser1().getCompleted().stream().anyMatch(course -> course.getId() == initializerService.getCourse1().getId()));
```

Comment:
In Tests immer über die Repositories abfragen, ob ein Entity gespeichert wurde. Andere **Objekte werden nicht automatisch upgedatet** nur, weil sie ein relationship haben.